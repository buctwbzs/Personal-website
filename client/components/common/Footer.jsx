import React from 'react'
import styled from 'styled-components'

const Foot = styled.footer`
  width: 100%;
  height: 68px;
  line-height: 68px;
  text-align: center;
  border-top: 1px solid #BFBFBF;
`


export default function Footer() {
  return (
    <Foot>
      Copyrights ©️ 2018 All Rights reserved by Buctwbzs.
    </Foot>
  )
}