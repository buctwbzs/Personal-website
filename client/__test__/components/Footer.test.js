import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '../../components/common/Footer'

it('Footer renders correctly', () => {
  const tree = renderer.create(<Footer />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})