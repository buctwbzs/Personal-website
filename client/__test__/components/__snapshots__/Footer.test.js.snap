// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="Footer__Foot-s17ddcxu-0 cvMZgF"
>
  Copyrights ©️ 2018 All Rights reserved by Buctwbzs.
</footer>
`;
